database.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+from settings import settings
```

Comment:
Ensure that the `settings` module is correctly configured and accessible. This is crucial for retrieving the `DATABASE_URL` from the environment or configuration file.